### DISCUSSION TITLE
is there going to be support for the Da Vinci Junior 1.0

### DISCUSSION THREAD

**Discussion by blackfire28** [luc-github/Repetier-Firmware-0.92#405] [F.A.Q] [answered]
the Da Vinci Junior 1.0 forces propriotary fillament and no one has made a fix.

**Comment by luc-github** [comment, marked as answer]
it is becaus they changed the hardware to use 2 chips, one for the FW and one for the control/monitoring of temperatures, make him porting very dificult
it is same board as pro actually.

this has been already explained here https://github.com/luc-github/Repetier-Firmware-4-Davinci/issues/287

and here https://github.com/luc-github/Repetier-Firmware-4-Davinci/issues/245#issuecomment-445457889

I do not have Davinci jr board and neither bandwidth for this to be honest
